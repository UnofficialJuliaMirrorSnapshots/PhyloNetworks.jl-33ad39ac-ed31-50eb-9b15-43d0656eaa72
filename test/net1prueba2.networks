((((6)#H-5:::0.6805144626250624,5):1.954268572767666,3):0.461227328281011,(2,(1,#H-5:::0.31948553737493757):2.478371059718378):0.6665298704988971,4);, with -loglik 128.4704561578712
(((5)#H-4:::0.5081358432996316,3):1.0463685044171078,(2,(1,(6,#H-4:::0.4918641567003684):4.598592963405185):0.037466519465998216):0.20087212270712623,4);, with -loglik 231.7736094851884
((3)#H-3:::0.5944208222904643,(2,(1,(6,(#H-3:::0.40557917770953567,5):9.893729510815922):1.0287950254466858):0.08886267733828704):0.10915585084821476,4);, with -loglik 215.41348384704173
(6,(((4)#H-2:::0.6100701427224106,3):0.029097601632317324,5):0.22046248932095286,(1,(2,#H-2:::0.38992985727758944):3.1965289740296914):1.1986265394207396);, with -loglik 172.22050695919188
((((6,(1,(2)#H8:::0.5694311834200659):5.411054959947867):0.15304140347881498,5):0.029744087155358624,3):1.6705680717951823,#H8:::0.43056881657993407,4);, with -loglik 192.03467244507002
