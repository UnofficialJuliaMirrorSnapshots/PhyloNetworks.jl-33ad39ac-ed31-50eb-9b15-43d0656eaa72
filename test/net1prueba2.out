((((6,#H7:::0.2856030435563703):10.0,5):0.5191607785499526,3):0.48941882463229214,(2,(1)#H7:::0.7143969564436297):1.5121336303781006,4); -Ploglik = 126.74250981974951
 Dendroscope: ((((6,#H7):10.0,5):0.5191607785499526,3):0.48941882463229214,(2,(1)#H7):1.5121336303781006,4);
 Elapsed time: 6.993290717 seconds in 5 successful runs
-------
List of estimated networks for all runs:
 (4,(((6,5):0.3353470932867715,3):0.12573320879015532,(1,(2)#H7:5.733697896500873::0.6269752705040429):0.8082647465545645):0.4002196144174887,#H7:0.07458340422944992::0.3730247294959571);, with -loglik 155.496257345519
 ((((6,#H7:::0.2856030435563703):10.0,5):0.5191607785499526,3):0.48941882463229214,(2,(1)#H7:::0.7143969564436297):1.5121336303781006,4);, with -loglik 126.74250981974951
 (((3,(5,(6)#H7:::0.6922870947532545):1.7390957703886551):0.4598961082497883,4):0.6610717818675633,(1,#H7:::0.3077129052467456):8.879027867903215,2);, with -loglik 128.2264279467482
 (3,((2,#H7:::0.32675875153374256):10.0,1):1.1071480539184226,((5,6):0.29503688508386916,(4)#H7:::0.6732412484662574):0.04619015937942773);, with -loglik 158.1387467189919
 ((3,(6,5):0.32545396986099545):0.0,(1,(2,#H7:::0.31783040568500653):9.999199878494535):1.1135649304712412,(4)#H7:::0.6821695943149935);, with -loglik 158.6226916101674
-------
